SUMMARY.md

```diff
@@ -21,6 +21,7 @@
   * [Vue.js](getting-started/quickstart-guides/vuejs.md)
   * [Next.js](getting-started/quickstart-guides/nextjs.md)
 * [How to use ImageKit.io?](getting-started/how-to-use-imagekit.io.md)
+* [Login for multi-account users](getting-started/login-for-multi-account-users.md)
```

Comment:
This should not be under `Getting started` section. It makes sense to have it under `Features`, `Collaboration` or `User access management` section.

Also this page should be called `Adding users to multiple accounts`.

Reply:
After having discussion, we have decided to title this page as `User in multiple accounts`

Reply:
`User associated with multiple accounts`

Reply:
done